src/services/phrases.service.ts

```diff
@@ -6,14 +6,27 @@ import polynomialService from '../services/polynomial.service';
 import { Polynomial } from '@/db/models/polynomial';
 import { sequelize } from '../db/models';
 import polynomialOptionService from './polynomial_option.service';
-
-const createPhrasesDB = (phrases: PhrasesCreationAttributes): Promise<PhrasesAttributes> => {
-  return Phrases.create(phrases, { raw: true }).then(({ id, text, group, polynomialId }) => ({
-    id,
-    text,
-    group,
-    polynomialId,
-  }));
+import surveyResultService from './survey_result.service';
+
+const createPhrasesDB = async (phrases: PhrasesCreationAttributes): Promise<PhrasesAttributes> => {
+  const phrase = await Phrases.create(phrases, { raw: true });
+  const polynomialOptions = await polynomialOptionService.getPolyOptionsFromPolyId(
+    phrase.polynomialId,
+  );
+  polynomialOptions.forEach(option => {
+    surveyResultService.createPhraseResults({
+      polynomialOptionId: option.id,
+      phraseId: phrase.id,
+      percentage: 0,
+    });
+  });
```

Comment:
Para evitar llamar tantas veces al servicio, utilizaría un bulkCreate que pase todos los datos que deben ser creados: https://sequelize.org/docs/v7/querying/insert/#inserting-in-bulk